.github/workflows/build.yaml

```
@@ -0,0 +1,225 @@
+---
+# GitHub action to compile and test kyua on supported platforms.
+#
+# Steps executed:
+# * Handle prerequisites
+#     * Install binary packages
+#     * Build packages from source (if needed).
+#     * Build
+#     * Install
+#     * Run tests
+#
+# On MacOS we build with:
+# * latest clang from brew (system provided clang lacks sanitizers).
+# * ld from system
+#
+env:
+    ATF_REPO: freebsd/atf
+    ATF_REF: atf-0.22
+    # LUTOK_REPO: freebsd/lutok
+    # LUTOK_REF: master
+    LUTOK_REPO: ngie-eign/lutok
+    LUTOK_REF: build-cleanup
+
+name: build
+on:
+    pull_request:
+        branches:
+            - master
+    push:
+    workflow_dispatch:
+
+permissions:
+    contents: read
+
+jobs:
+    build:
+        name: Build ${{ matrix.build-os }} ${{ matrix.compiler }} ${{ matrix.enable-atf }}
+        runs-on: "${{ matrix.build-os }}"
+        strategy:
+            fail-fast: false
+            matrix:
+                build-os:
+                    - ubuntu-24.04
+                    - macos-latest
+                #enable-atf: [--disable-atf, --enable-atf]
+                enable-atf: [--enable-atf]
+                include:
+                    - build-os: macos-latest
+                      compiler: clang-19
+                      pkgs:
+                          - llvm@19
+                          - autoconf
+                          - automake
+                          - libtool
+                          - pkgconf
+                          - lua@5.4
+                          - sqlite
+                      enable-atf-pkgs:
```

Since the below line is commented out, add brackets to make this valid YAML
```suggestion
                      enable-atf-pkgs: []
```